### PR TITLE
#582 Appveyor fails when using specific features of Java 7

### PR DESCRIPTION
**Pull request by 2686747**
#582 fix

### CODE REVIEW

**Comment by davvd** [comment]
@2686747 Many thanks for the PR, let me find a reviewer for it

**Comment by davvd** [comment]
@hdouss please review, thanks

File: pom.xml [hunk 1]
```diff
@@ -351,8 +360,8 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-compiler-plugin</artifactId>
                         <configuration>
-                            <source>1.7</source>
-                            <target>1.7</target>
+                            <source>${jdk.version}</source>
```
Comment (HDouss) [inline pom.xml]: @2686747 We could remove all this part. Why should we have a special compiler configuration for qulice profile ?

Comment (2686747) [inline pom.xml]: @HDouss Thanks, sure, removed.

**Comment by HDouss** [comment]
@2686747 Thanks

**Comment by HDouss** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by davvd** [comment]
@ypshenychka please, review this issue for QA compliance, as per [par.24](http://at.teamed.io/policy.html#24)

**Comment by davvd** [comment]
@rultor please deploy

**Comment by ypshenychka** [comment]
@2686747 
>#582 fix

According to our [QA Rules](http://at.teamed.io/qa.html):
>Pull request description explains the solution proposed and contains a link to the original ticket it is related to.

Please provide an appropriate description in compliance with above rule.

**Comment by ypshenychka** [comment]
@HDouss 
According to our [QA Rules](http://at.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by HDouss** [comment]
@ypshenychka I confirm.

**Comment by ypshenychka** [comment]
@HDouss Thanks.

**Comment by 2686747** [comment]
@ypshenychka Thanks, it's fixed

**Comment by ypshenychka** [comment]
@2686747 Thank you.

**Comment by ypshenychka** [comment]
@davvd Quality is acceptable.

**Comment by HDouss** [comment]
@davvd Could you please pay me here, and increase my rating accordingly ?

**Comment by 2686747** [comment]
@davvd same question

**Comment by davvd** [comment]
> @davvd Quality is acceptable.

@ypshenychka thanks for the review, we'll try better next time

**Comment by davvd** [comment]
@hdouss I added 10 mins to @ypshenychka (for QA review) in transaction 78660891. thanks, paid, **15 mins** to your account, payment ID is `78660902`, task took 29 hours and 20 mins to complete. +15 added to your rating, at the moment it is: [+3081](http://www.netbout.com/b/36032?open=rating)

**Comment by 2686747** [comment]
@davvd could you, please, calculate it for me?

**Pull request merged**